hibernate-core/src/main/java/org/hibernate/internal/AbstractSharedSessionContract.java

```diff
@@ -389,6 +392,20 @@ public boolean isTransactionInProgress() {
 		return !isClosed() && transactionCoordinator.isTransactionActive();
 	}
 
+	@Override
+	public void checkTransactionNeededForUpdateOperation(String exceptionMessage, boolean convertException) {
```

Comment:
I don't think it should be necessary to have the ```convertException``` argument.

Please see https://github.com/hibernate/hibernate-orm/pull/2780/files#r258717383